src/child/components/Favourite.js

```diff
@@ -76,11 +91,33 @@ class Favourite extends Component {
         }
     }
 
+    onModalClick(e) {
+        e.stopPropagation();
```

Comment:
`stopPropagation` is best avoided (and in the other instances that it's used in this PR), would it be easy to use an alternative (`preventDefault`)?